README.md

```diff
@@ -35,7 +32,8 @@ Target users/projet :
   * spyking-circus team : handle probe with this package
   * kilosort team : handle probe with this package
   * tridesclous team : handle probe with this package
- 
+  * open ephys team : automatically generate channel map configuration files
+
```

Comment:
yeah!! Do you plan to use this format ?

Reply:
Yep! We current use JSON for our [Channel Map](https://open-ephys.github.io/gui-docs/User-Manual/Plugins/Channel-Map.html) plugin, so it will be easy to integrate the probeinterface format

Reply:
Maybe you extend this plugin to rely on probeinterface json format and so benefit the probe library.

Reply:
Yes, that's the plan!

Reply:
Cool.